src/utils/igvBrowserHelper.ts

```diff
@@ -2,6 +2,7 @@ import { Dispatch } from 'react';
 import { Browser } from 'igv';
 
 export const updateQueryParams = (key: string, value: string) => {
+  if (value === 'viphog') return;
```

Comment:
If I understand right, this is here to ignore changes to the virify track when setting the query params? If so, might it be better to instead check for whether the track IS the functional annotation one? In case we add some other track in future (like an explicit crispr track or something), that would then interfere?

Reply:
This was an oversight. 
It was from when I was initially doing some testing. 
I've made adjustments to generally update the query params, only for functional annotations. 
This way any extra annotation filters will not interfere.